src/utils.rs

```diff
@@ -110,3 +112,15 @@ fn verify_zpad_exact() {
 fn verify_zpad_less_than_size() {
     assert_eq!(zpad(&[1, 2, 3, 4], 2), [1, 2, 3, 4]);
 }
+
```

Comment:
why do we need this? didn't we just remove bigendianint?

Reply:
This how we define how `Uint256` will serialize the data